roles/openshift_logging/tasks/build_patch.yaml

```diff
@@ -0,0 +1,58 @@
+---
```

Comment:
I have some hesitation to implement something like this.  Why is building such a complicated set of patch files necessary?

If anything, this looks like it should be created as an action plugin, and then we register the output of that plugin to create whatever file/patch we need.

As-is, this is pretty hard to follow.

Reply:
There is a requirement to maintain any custom configuration changes to currently deployed configmap files for logging while still having a way to patch in any necessary changes that the installer may need to make to allow our images to work.

I could try to rewrite this as a plugin instead.